source/strings.tex

```diff
@@ -1218,21 +1216,16 @@
 \begin{itemdescr}
 \pnum
 \effects
-Constructs an object of class
-\tcode{basic_string}
-as indicated in Table~\ref{tab:strings.ctr.cpy}.
+Constructs an object of class \tcode{basic_string}.
+
+\pnum
+\postconditions
+\tcode{data()} points at the first element of an allocated copy of the
+array whose first element is pointed at by \tcode{str.data()},
+\tcode{size()} is equal to \tcode{str.size()}, and \tcode{capacity()} is
```

Comment:
I'm not sure this is right. `size()` is equal to the value that `str.size()` had prior to the call, right? `str.size()` is unspecified. (Likewise for `data()`.) I think we're usually more explicit about whether we're talking about the before / after value.

Reply:
Would it help if I added "the original value of" in a few places?

Reply:
@zygoloid: "original value" is what the allocator versions already use. In fact, my attempt to add "original value" to the description here led me to independently invent *identical* wording!